include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -668,7 +667,7 @@ __parallel_transform_scan_single_group(_ExecutionPolicy&& __exec, _InRng&& __in_
                                        ::std::size_t __n, _UnaryOperation __unary_op, _InitType __init,
                                        _BinaryOperation __binary_op, _Inclusive)
 {
-    using _CustomName = typename std::decay_t<_ExecutionPolicy>::kernel_name;
+    using _CustomName = oneapi::dpl::execution::policy_kernel_name<_ExecutionPolicy>;
```

Comment:
In some places `oneapi::dpl::execution::policy_kernel_name<_ExecutionPolicy>` is used, in others `oneapi::dpl::execution::__dpl::policy_kernel_name<_ExecutionPolicy>` is used.  

Which is correct? (As implemented, it seems like the second one...)  Should this be made consistent?

Reply:
Fixed